pyvista/themes.py

```diff
@@ -2989,7 +2990,24 @@ def lighting_params(self, config: _LightingConfig):
         self._lighting_params = config
 
 
-class DarkTheme(DefaultTheme):
+class DefaultTheme(Theme):
+    """Deprecated default theme.
+
+    .. deprecated:: 0.40.0
+        Deprecated an renamed to ``Theme``.
+
+    """
+
+    def __init__(self):
+        """Initialize the theme."""
+        super().__init__()
+        warnings.warn(
+            '`DefaultTheme` has been deprecated and renamed `Theme`. Further, it is no longer the default theme.',
```

Comment:
"DefaultTheme is deprecated and rename Theme.  DefaultTheme is no longer the default theme, nor is Theme the default theme."  The warning message here needs work, although I don't have a better suggestion right now.

Reply:
Haha, I spent a few minutes thinking on this and I feel it is clear and concise. I'm open to suggestions to improve the message here.